rules/windows/sysmon/sysmon_registry_trust_record_modification.yml

```diff
@@ -0,0 +1,24 @@
+title: Windows Registry Trust Record Modification
+id: 295a59c1-7b79-4b47-a930-df12c15fc9c2
+status: experimental
+description: Alerts on trust record modification within the registry, indicating usage of macros
+references:
+    - https://outflank.nl/blog/2018/01/16/hunting-for-evil-detect-macros-being-executed/
+    - http://az4n6.blogspot.com/2016/02/more-on-trust-records-macros-and.html
+author: Antonlovesdnb
+date: 2020/02/19
+modified: 2020/02/19
+tags:
+    - attack.initial_access
+    - attack.t1193
+logsource:
+    product: windows
+    service: sysmon
+detection:
+    selection:
+        EventID: 12
+        TargetObject|contains: 'TrustRecords'
+    condition: selection
+falsepositives:
+    - Alerts on legitimate macro usage as well, will need to filter as appropriate
+level: medium

```

Comment:
How often does this trigger? On every execution of macros?

Reply:
Going off this post ( http://az4n6.blogspot.com/2016/02/more-on-trust-records-macros-and.html ) the registry key will be modified when a user has clicked the enabled editing / enable content on word docs containing macros from an untrusted location. In my testing, if a user had already clicked on enable editing / enable content on a document, this registry key will not be modified; so this will not trigger on every execution of macros, just one's that the user has clicked enable content / enable editing on.

Reply:
Ah, okay - I'd say 'medium' is okay